server/routes/contacts/create/createContactRoutes.ts

```diff
@@ -0,0 +1,53 @@
+import { Router } from 'express'
+import AuditService from '../../../services/auditService'
+import SuccessController from './success/createContactSuccessController'
+import logPageViewMiddleware from '../../../middleware/logPageViewMiddleware'
+import EnterNameController from './enter-name/createContactEnterNameController'
+import { validate } from '../../../middleware/validationMiddleware'
+import { createContactEnterNameSchemaFactory } from './enter-name/createContactEnterNameSchemas'
+import CreateContactEnterDobController from './enter-dob/createContactEnterDobController'
+import { createContactEnterDobSchema } from './enter-dob/createContactEnterDobSchemas'
+import StartCreateContactJourneyController from './start/startCreateContactJourneyController'
+import ensureInCreateContactJourney from './createContactMiddleware'
+
```

Comment:
I like this - all of the routes together for creating a contact. This is more like BVLS method, so I find this more familiar and easier to follow.